docs/development/glossary/index.rst

```diff
@@ -53,6 +53,11 @@ or Client OS be used instead.
 minidump_stackwalk program are stored. The actual files are stored
 with a .jsonz extension.
 
+**Priority job**: A reprocessing request (see ``Reprocessing``) that has
```

Comment:
priority jobs only happen to crashes that have never been processed, so it is not a "reprocessing" request.